src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1184,14 +1184,33 @@ Prevent Long URL From Breaking Out of Container
     margin-right: 5px;
 }
 
-/*
-Glyphicon for success status messages
-*/
+/*//////////////////////////////////////////////////////////
+////////////////////// Status Icons //////////////////////
+//////////////////////////////////////////////////////////*/
 
-.icon-success:before {
+.icon-success:before,
+.icon-warning:before,
+.icon-danger:before,
+.icon-info:before {
+    /*noinspection CssNoGenericFontName*/
```

Comment:
> There's a reason why IntelliJ (and other IDEs) made their inspection suppressions as comments, so they become just comments outside of the IDE.

They aren't really the same as normal comments. For one thing, they make code less readable rather than more readable.

Agree with your points about no throwing away good features; however, my main point is that to introduce such specific suppressions, we'd need to come to a decision which IDE default inspections we'd like to adopt, then consistently follow them.

> As for whether we treat the default settings as authoritative, that's up to the project?

Exactly - the problem is, our project doesn't have any such recommendation right now. These things are considered to be user-specific.

> We should turn those inspections off completely if we don't agree with it, not ignore them and leave the warnings hanging around.

People who use Eclipse or don't use an IDE won't be bothered by them (instead, they'd be bothered by unnecessary comments that are suppressions, and wouldn't have any option to hide them). People who use IntelliJ and are bothered by warnings have the options of turning off the specific inspections that generate those warnings, or living with the warnings.

To sum up, these things are all part of the specific local setup that we don't dictate. I, too, am an IntelliJ fan. However, adding stuff to the codebase (comments or otherwise) for the express purpose of conforming to a certain IDE is what I'm against. I'll only have it if we specifically decide to elevate one IDE's authority over the others, then consistently stick to it throughout the codebase. Doing so would indeed bring about benefits, though at the cost of additional burden to document and enforce these recommendations (e.g. all reviewers would have to open the code in IntelliJ to check that it didn't trigger another warning, then tell the developer to add in a specific suppression comment before approving the PR).